### PR TITLE
[Tests][Cypress]: Create Org Bounty from Organization Page

### PR DESCRIPTION
**Pull request by aliraza556**
## Describe your changes
This PR adds the Cypress test for  `Create org bounty from organization page`.

Closes: #424

## Issue ticket number and link:
- **Ticket Number:** [ 424 ]
- **Link:** [ https://github.com/stakwork/sphinx-tribes-frontend/issues/424 ]

## Type of change

Please delete options that are not relevant.
- [ ] New feature (non-breaking change which adds functionality)

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device

### CODE REVIEW

**Comment by aliraza556** [comment]
Hi @elraphty,
Please give me just 10 minutes; I will push the new changes.

**Comment by elraphty** [comment]
> Hi @elraphty, Please give me just 10 minutes; I will push the new changes.

Okay, please also link the issue.

**Comment by aliraza556** [comment]
> > Hi @elraphty, Please give me just 10 minutes; I will push the new changes.
> 
> Okay, please also link the issue.

Yeah

File: cypress/e2e/24_createOrgPageBounty.cy.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    cy.contains(org.name);
+    cy.wait(1000);
+
+    cy.contains(org.name).invoke('removeAttr', 'target').click({ force: true });
+    cy.wait(5000);
+
+    cy.url().should('include', '/org/bounties');
+    cy.wait(5000);
+
```
Comment (elraphty) [inline cypress/e2e/24_createOrgPageBounty.cy.ts:63]: @aliraza556 Why add this here when you already have it in the create org bounty function?

File: cypress/e2e/24_createOrgPageBounty.cy.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    cy.contains('Post a Bounty').click();
```
Comment (elraphty) [inline cypress/e2e/24_createOrgPageBounty.cy.ts:64]: @aliraza556 Why add this here when you already have it in the create org bounty function?

Comment (elraphty) [reply cypress/e2e/24_createOrgPageBounty.cy.ts:64]: The `create_orgBounty` also searches for the `Post a Bounty` button.

Comment (aliraza556) [reply cypress/e2e/24_createOrgPageBounty.cy.ts:64]: @elraphty,
Just Verify this point: 
image

Comment (aliraza556) [reply cypress/e2e/24_createOrgPageBounty.cy.ts:64]: @elraphty If you don't want this, then I will remove these lines of code.

Comment (elraphty) [reply cypress/e2e/24_createOrgPageBounty.cy.ts:64]: > @elraphty If you don't want this, then I will remove these lines of code

> @elraphty, Just Verify this point: image
> 
> image

Oh! perfect, I see now.

**Pull request merged**